lib/server-code/runners/cloud-workers-broker.js

```diff
@@ -73,9 +74,26 @@ class WorkersBroker extends EventEmitter {
     setInterval(() => {
       getAllWorkers().forEach(worker => {
         if (worker.heartbeat + this.heartbeatTimeout < Date.now()) {
+          const { appId, task } = worker
+          const details = [`appId: ${appId}`]
+
+          if (task.method) {
+            details.push(`method: ${task.method}`)
+          }
+
+          if (task.deploymentModelName) {
+            details.push(`deployed service: ${task.deploymentModelName}`)
+          }
+
+          if (task.eventId) {
+            const event = ServerCodeEvents.get(task.eventId)
+            details.push(`handler-event-name: ${event.provider.id}(${event.name})`)
+          }
+
```

Comment:
Move everything from 78 to 92 lines to separate function e.g. `getKilledWorkerDetails` at the top of file, and use it like this:
```js
const workerDetails = getKilledWorkerDetails(worker)

this.killWorker(worker, (
      `Worker expired due to heartbeat timeout (${this.heartbeatTimeout}ms), ` +
      `the last ping was: ${worker.heartbeat && new Date(worker.heartbeat).toISOString()}`
      `the last ping was: ${worker.heartbeat && new Date(worker.heartbeat).toISOString()} ` +
      `${workerDetails}`
))
```